config/jobs/kubernetes-sigs/kjob/kjob-periodics-release-0.1.yaml

```diff
@@ -0,0 +1,124 @@
+periodics:
+  - interval: 12h
+    name: periodic-kjob-test-unit-release-0-1
+    cluster: eks-prow-build-cluster
+    annotations:
+      testgrid-dashboards: sig-scheduling
```

Comment:
Is this the right sig? Is sig-apps sponsoring this?

Reply:
Looks like we have sig-apps for the testgrid dashboard throughout. We should change this so I won't block on this at the moment. But we should change this to be sig-apps.

Reply:
Updated. PTAL.